### DISCUSSION TITLE
2024 contribution

### DISCUSSION THREAD

**Discussion by ZYMNZ** [community/community#83640] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello there,
I noticed that whenever I want to view my 2024 contributions, it doesn't display it directly until I refresh the page! I don't know if this is a bug from my side only. 

![image](https://github.com/community/community/assets/98342638/d33338a2-916f-43c3-ad97-47da1f8b3949)

**Comment by kshetritej** [comment]
Well it's not bug for sure.

**Comment by muxalik** [comment]
That's how it's supposed to work)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ZYMNZ ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
